src/dependencies.jl

```diff
@@ -199,6 +199,30 @@ pkg_name(p::Pacman) = p.package
 
 libdir(p::Pacman,dep) = ["/usr/lib", "/usr/lib32"]
 
+# zypper is a package manager used by openSUSE
+const has_zypper = try success(`zypper --version`) catch e false end
+type Zypper <: PackageManager
+    package::AbstractString
+end
+can_use(::Type{Zypper}) = has_zypper && OS_NAME == :Linux
+package_available(z::Zypper) = can_use(Zypper) && success(`zypper se $(z.package)`)
+function available_version(z::Zypper)
+    uname = readchomp(`uname -m`)
+    found_uname = false
+    found_version = false
+    for l in eachline(`zypper info $(z.package)`)
```

Comment:
ref https://github.com/JuliaLang/BinDeps.jl/issues/188#issuecomment-161597464, I think we want to do something like
```
ENV2 = copy(ENV)
ENV2["LC_ALL"] = "C"
for l in eachline(setenv(`zypper info $(z.package)`, ENV2))
```

And what are `found_uname` and `found_version` being used for here?